### ISSUE TITLE
Too small a chance of dragon_upgrade_smithing_template sleeping.

### ISSUE THREAD

**Issue by Foxy1e3t** [Matsulen/LemonKindom-Artifacts#4]
I flew a total of more than 300 thousand blocks in Ender World and for all that time found only one template. Judging by the loot tables it spawns also in hell, but there too I found nothing. By my stupidity I spent this one template to improve the breastplate, and to give myself another one I think unfair. 
By the way, I also didn't find a single Crystal Original Plate in the entire game.

**Comment by Matsulen** [comment]
Thank you for your comment, I’ve got your feedback.
I will soon check the loot chance and also fix the template problem. I
promise there will be an update soon, also thank you for spending time on
playing my first mod.
Basically I’m also developing a new mod focus on new structures spawn in
world in order to add more loots about templates and evolve materials, this
will also been release soon. Hope you enjoy them and Thank you again for
your valuable feedback.

Sincerely,
Matsulen

Foxy1e3t ***@***.***>于2023年12月30日 周六0:55写道：

**Comment by Matsulen** [comment]
By the way for the version right now, you can also find Dragon Template from the nether world.
I'm updating and I will also add more chest loot for both template and Crystal Original Plate.

**Comment by Foxy1e3t** [comment]
What about creating a wiki or something similar to understand where to get this or that item? The mod is very interesting and the textures of the items are very nice, I would like more animations. The mod definitely deserves attention!

**Comment by Matsulen** [comment]
Thank you for your valuable feedback. Currently a wiki for me is quite a big workload, but I will consider it in the future.
More animations are also including in future updates, I really hope you can like it.

**Comment by Foxy1e3t** [comment]
Maybe I could make a wiki for this mod? Only for that I need information on all the subjects

**Comment by Matsulen** [comment]
Yes! Of course you can. It still needs some time for me to clear up all the informations about those subjects.
